mzPlot/Plots/ScatterPlot.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BayesianEstimation;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using MzLibUtil;
+using OxyPlot;
+using System.Linq;
+using OxyPlot.Series;
+using OxyPlot.Axes;
+
+namespace mzPlot
+{
+    public class ScatterPlot : Plot
+    {
+        /// <summary>
+        /// Creates a scatter plot. The data X value is the X-coordinate, the data Y value is the Y-coordinate.
+        /// </summary>
+        public ScatterPlot(OxyPlot.Wpf.PlotView oxyPlotView, IEnumerable<Datum> data, OxyColor? markerColor = null, double markerSize = 3,
```

Comment:
what about x- and y-error bars?

can axes be on log scale?

Reply:
both probably in a future PR